soc/wch/ch32v/qingke_v2a/vector.S

```diff
@@ -15,7 +15,7 @@ GTEXT(__initialize)
 SECTION_FUNC(vectors, ivt)
 	.option norvc
 	j       __start
-	.rept 38
+	.rept   38
```

Comment:
this line change should be in another commit

Reply:
Sorry for my late reply. Done.